README.md

```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
 While adopting kafka at real.digital we noticed the immense entry barrier it poses to newcomers. 
 We can't recount how often we wrote Slack messages asking for the script to check the 
 status of topics or consumer groups. This is partly (but not only) due to a 
-fragmented and unclear definition of tooling and APIs for kafka. 
-In a wide array of administration tools, `esque` distances itself by striving to provide Kafka Ops for Humans, in a usable and natural way. We feel that the goal of `esque` embodies one of the Perl design principles: “**keep easy things easy, and make hard things possible**”. 
+fragmented and unclear definition of tooling and APIs for kafka.
+In a wide array of administration tools, `esque` distances itself by striving to provide Kafka Ops for Humans, in a usable and natural way.    
+  
+We feel that the goal of `esque` embodies the principle: “**keep easy things easy, and make hard things possible**”. 
 
 ## Principles
 
-Our driving objectives can be summed up through these principles:
-* comes with batteries included
+* batteries included
 * feature rich
 * robust
 * insightful
-* made by engineers for engineers
+* by engineers for engineers
 
-## Basic Features
+## Feature Overview
 
 * Support for any type of Kafka deployment >1.2
-* List Resources (Topics, Consumergroups, Brokers)
+* Listing of Resources (Topics, Consumergroups, Brokers)
 * Get detailed Overviews of Resources (Topics, Consumergroups, Brokers)
 * Create/Delete Topics
+* Edit Topic Configurations
+* Edit Consumer Offset for Topics
+* SASL/SSL Support support out of the box
+* Consume and Produce to and from Avro and Plaintext Topics (including Avro Schema Resolution from Schema Registry)
 * Context Switch (Easily Switch between pre-defined Clusters)
 * Kafka Ping (Test roundtrip time to your kafka cluster)
 
-## Installation and Usage
+## Command Overview
 
-### Installation
+```bash
+$ esque
```

Comment:
Usage: esque [OPTIONS] COMMAND [ARGS]...

  esque - an operational kafka tool.

  In the Kafka world nothing is easy, but esque (pronounced esk) is an
  attempt at it.

Options:
  --recreate-config  Overwrites the config with the sample config.
  --version          Show the version and exit.
  -v, --verbose      Return stack trace on error.
  --no-verify        Skip all verification dialogs and answer them with yes.
  --help             Show this message and exit.

Commands:
  apply     Apply a set of topic configurations.
  config    Configuration-related options.
  consume   Consume messages from a topic.
  create    Create a new instance of a resource.
  ctx       List contexts and switch between them.
  delete    Delete a resource.
  describe  Get detailed information about a resource.
  edit      Edit a resource.
  get       Get a quick overview of different resources.
  ping      Test the connection to the kafka cluster.
  produce   Produce messages to a topic.